site/app/controllers/pdf/PDFController.php

```diff
@@ -172,13 +172,13 @@ public function savePDFAnnotation(string $gradeable_id, string $target_dir) {
         return true;
     }
 
-    public function getAnonPath($file_path) {
+    public function getAnonPath($file_path, $g_id) {
         $file_path_parts = explode("/", $file_path);
         $anon_path = "";
         for ($index = 1; $index < count($file_path_parts); $index++) {
             if ($index === 9) {
                 $user_id = $file_path_parts[$index];
-                $anon_ids = $this->core->getQueries()->getUserFromAnon($user_id);
+                $anon_ids = $this->core->getQueries()->getUserFromAnon($user_id, $g_id);
```

Comment:
Should we consider anon team id here

Reply:
Looking at some related code, it feels like it should be there. Surprising that no unexpected behavior was ever reported due to this on production machine.